src/test/java/org/folio/rest/support/WebClient.java

```diff
@@ -5,30 +5,32 @@
 import java.net.URL;
 import java.util.UUID;
 
+import io.vertx.core.AsyncResult;
 import io.vertx.core.Handler;
 import io.vertx.core.Vertx;
-import io.vertx.core.http.HttpClientRequest;
-import io.vertx.core.http.HttpClientResponse;
+import io.vertx.core.buffer.Buffer;
 import io.vertx.core.json.Json;
 import io.vertx.core.logging.Logger;
 import io.vertx.core.logging.LoggerFactory;
+import io.vertx.ext.web.client.HttpRequest;
+import io.vertx.ext.web.client.HttpResponse;
 
-public class HttpClient {
+public class WebClient {
```

Comment:
Why has this class been renamed?

Reply:
According to https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-271 , HttpClient is deprecated in with WebClient.

This test utility class was simply helping provide HttpClient.
Now that it is providing WebClient, it makes more sense to me to update the name to communicate to the coder what this library is now wrapping and using.

Reply:
> According to https://github.com/folio-org/raml-module-builder/blob/master/doc/upgrading.md#version-271 , HttpClient is deprecated in with WebClient.

I think that guidance refers to changes in vert.x, where some methods from HttpClient were deprecated. Maybe RAML Module Builder is advocating migration from vert.x HttpClient to WebClient?

> This test utility class was simply helping provide HttpClient.
> Now that it is providing WebClient, it makes more sense to me to update the name to communicate to the coder what this library is now wrapping and using.

I think I interpret the intent of this class differently. Rather than intended to provide the underlying vert.x HttpClient, I thought it was trying to hide it away from the rest of the test code.

Personally, I'd prefer that the client test code was ignorant as to whether the internal implementation is using vert.x HttpClient, WebClient or a different library.

Could we come up with a name that better describes the intent of this wrapper? In mod-circulation, it is called OkapiHttpClient (in production code, most of the test code has been ported to using RestAssured).

Reply:
I feel like Okapi is completely designed around the HTTP protocol.
Would it be fine with shortening `OkapiHttpClient` to `OkapiClient`, or is that too generic?
Either way, I agree that using `OkapiHttpClient` or `OkapiClient` is an improvement over `HttpClient` or `WebClient`.

Reply:
> I feel like Okapi is completely designed around the HTTP protocol.

> Would it be fine with shortening `OkapiHttpClient` to `OkapiClient`, or is that too generic?
> Either way, I agree that using `OkapiHttpClient` or `OkapiClient` is an improvement over `HttpClient` or `WebClient`.

I'm ok with either `OkapiClient` or `OkapiHttpClient`.

I tend to prefer more specific names and it sets the context for the method names.

Reply:
Thanks for renaming this.